src/pages/docs/kubernetes/live-object-status/troubleshooting/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2025-03-28
+modDate: 2025-03-28
+navTitle: Troubleshooting
+title: Troubleshooting
+navSection: Troubleshooting
+description: How to troubleshoot common Kubernetes Live Object Status issues
+navOrder: 70
+---
+
+This page will help you diagnose and solve issues with Kubernetes Live Object Status.
+
+## Installation Issues
+
+### gRPC port 8443
+
+blah blah
```

Comment:
We probably need to complete this page before we push to production 😆

Reply:
Apologies Steve, this is really a draft still